iron-menu-behavior.html

```diff
@@ -23,7 +23,8 @@
   Polymer.IronMenuBehaviorImpl = {
 
     properties: {
-
+      _searchText: '',
```

Comment:
or if you want me to start with the class without it, that's fine, but:

a) I've heard changing shapes are bad mmmkay
b) it's harder to tell when there are programming errors with more type variance

i.e. if `this._searchText === undefined` is it because it's uninitialized or the method was called with the wrong `this` value?